test/frontend.js

```diff
@@ -24,7 +24,6 @@ function login (username) {
   return function (n) {
     n
       // Logout
-      .click(elT('openProfileDropDown'))
       .click(elT('logoutBtn'))
```

Comment:
@taoeffect and @hubudibu I have a question. The test failing is
> 1) Frontend Sign up Test Test Logout and Login:
> Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.

I suspect the problem is inside this login function.

`'openProfileDropDown'` doesn't exist anymore, now it's only `'logoutBtn'`, so I've removed that line.

But the test is still failing. Can you explain me better why the first step of `login()` is to Logout? Can I better debug these tests? Do breakpoints and see the browser at that exact moment, etc...

Reply:
You can run the tests and see them visually:

```
$ SHOW_BROWSER=1 grunt test
```

And you can use Mocha's `.only` and `.skip` modifiers to only run certain tests and skip others (see the Mocha documentation).

You can also insert purposeful `.wait(1000)`s to slow down the tests.